src/components/InfiniteHits.vue

```diff
@@ -3,11 +3,28 @@
     v-if="state"
     :class="suit()"
   >
+
+    <slot
+      v-if="state.widgetParams.showPrevious"
+      name="loadPrevious"
+      :refinePrevious="refinePrevious"
+      :page="state.results.page"
+      :is-first-page="state.isFirstPage"
+    >
+      <button
+        :class="[suit('loadPrevious'), state.isFirstPage && suit('loadPrevious', 'disabled')]"
+        :disabled="state.isFirstPage"
+        @click="refinePrevious()"
+      >Show previous results</button>
+    </slot>
+
     <slot
       :items="items"
       :results="state.results"
       :is-last-page="state.isLastPage"
-      :refine="refine"
+      :refinePrevious="refinePrevious"
+      :refineNext="refineNext"
+      :refine="refineNext"
```

Comment:
I wonder if we instead should have `refine(+1)` and `refine(-1)` as the API (with fallback on `refine() === refine(+1)`

Reply:
Talked about it with @samouss during the RFC review ([discussion](https://github.com/algolia/instantsearch-rfcs/pull/16#discussion_r270315733)). I had `refine('previous')` and `refine('next')` but we agreed for `refineNext()` and `refinePrevious()`.

But we can re-discuss it together :wink: